client/src/components/layout/ReviewTile.js

```diff
@@ -2,7 +2,24 @@ import React from 'react'
 import Vote from "./Vote.js"
 
 const ReviewTile = (props) => {
-  const { id, heading, description, rating, voteCount } = props.review
+  const {heading, description, rating, id, voteCount } = props.review
+  
+  const deleteReview = () => {
+    if (props.user) {
+      if (props.user.id === props.review.userId) {
+        return(
+          <button onClick={handleDeleteClick}> 
+            Delete Review
+          </button>
+        )
+      }
+    }
+  }
```

Comment:
`deleteReview` vs `DeleteYourReview` is starting to get confusing. I would rename this to be `deleteButtonElement`

Further, this doesnt need to be in a function at all. 

Instead, create a variable `deleteButtonElement` that is set to an empty string to begin. then

```js
if (props.user) {
      if (props.user.id === props.review.userId) {
         deleteButtonElement = <button onClick={handleDeleteClick}> 
            Delete Review
          </button>
      }
    }
```

Finally, add `deleteButtonElement` to your `ReviewTile` return